src/components/reusable/errorComponent/errorComponent.ts

```diff
@@ -0,0 +1,63 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import ErrorComponentScss from './errorComponent.scss';
+
+import '@kyndryl-design-system/shidoka-foundation/components/button';
+
+/**
+ * Error component.
+ * @slot image - Slot for error image.
+ * @slot description - Slot for error description.
```

Comment:
The description could use the unnamed slot, since there isn't one yet.

Reply:
done